Sources/PinLayout+WrapContent.swift

```diff
@@ -94,14 +101,22 @@ extension PinLayout {
      - Parameters:
      - type: Specify the wrap type (.all, .horizontally, .vertically)
      - padding: Specify a padding using an UIEdgeInsets.
+     - viewFilter: Specify whether to include all views or only visible ones.
      */
     @discardableResult
-    public func wrapContent(_ type: WrapType, padding: PEdgeInsets) -> PinLayout {
-        return wrapContent(type, padding: padding, { return "wrapContent(\(type.description), padding: \(insetsDescription(padding))" })
+    public func wrapContent(_ type: WrapType, padding: PEdgeInsets, viewFilter: ViewFilter = .none) -> PinLayout {
+        return wrapContent(type, padding: padding, viewFilter: viewFilter, { return "wrapContent(\(type.description), padding: \(insetsDescription(padding))" })
     }
 
-    private func wrapContent(_ type: WrapType, padding: PEdgeInsets, _ context: Context) -> PinLayout {
-        let subviews = view.subviews
+    private func wrapContent(_ type: WrapType, padding: PEdgeInsets, viewFilter: ViewFilter = .none, _ context: Context) -> PinLayout {
+        let subviews: [PinView.PinView]
+        switch viewFilter {
+        case .visibleOnly:
+            subviews = view.subviews.filter { !$0.isHidden }
```

Comment:
What about views with `alpha = 0`?

There is public `visible(_ views: [UIView])` views filter that takes alpha into account - https://github.com/layoutBox/PinLayout/blob/babea9c8e821dfb0687e66c123fed2006e688929/Sources/Filters.swift#L25, it may lead to confusion if name "view filter" will behave differently here.

Reply:
Good point, you could use the `visible()` here to filter views.